app/models/concerns/soft_deletable.rb

```diff
@@ -4,7 +4,7 @@ module SoftDeletable
   included do
     default_scope { not_deleted }
     scope :deleted, -> { unscoped.where.not(deleted_at: nil) }
```

Comment:
ici j'arrive pas a enlever le `unscoped` parce que si je l'enlève ça équivaut à `where(deleted_at: nil)..where.not(deleted_at: nil)` ce qui donne rien

Reply:
par contre dès qu'on appel ce scope ça supprime tout les scope qui étaient appelé en même temps, filtres etc et ça prend toutes les instances du modele